### PR TITLE
refactor(ui):restore task details and message record details UI,css3 animation

### PR DESCRIPTION
**Pull request by wzh425**

### CODE REVIEW

File: src/Contracts/Masa.Mc.Contracts.Admin/Enums/MessageTasks/MessageTaskHistoryStatuses.cs [hunk 1]
```diff
@@ -8,5 +8,8 @@ public enum MessageTaskHistoryStatuses
     WaitSend = 1,
     Sending,
     Completed,
-    Withdrawn
+    Withdrawn,
+    Success = 101,
```
Comment (MayueCif) [inline src/Contracts/Masa.Mc.Contracts.Admin/Enums/MessageTasks/MessageTaskHistoryStatuses.cs:12]: 1 101?

File: src/Web/Masa.Mc.Web.Admin/Pages/MessageTasks/Modules/OrdinaryMessageInfo.razor [hunk 2]
```diff
@@ -1,51 +1,29 @@
...
-                Label="@T("DisplayName.MessageTemplateJumpUrl")"
-                Outlined
-                Disabled
-                HideDetails="@("auto")">
-        </MTextField>
-    </MRow>
-}
\ No newline at end of file
+<div>
+    <dl class="mt-8">
```
Comment (MayueCif) [inline src/Web/Masa.Mc.Web.Admin/Pages/MessageTasks/Modules/OrdinaryMessageInfo.razor:3]: mt-8 change to parent pt-8

**Review by MayueCif — COMMENTED** [review]
@wzh425

**Review by MayueCif — APPROVED** [review]
@doddgu LGTM

File: src/Web/Masa.Mc.Web.Admin.Server/Pages/_Layout.cshtml [hunk 3]
```diff
@@ -17,10 +17,12 @@
     <link href="https://cdn.masastack.com/npm/@("@mdi")/font@5.x/css/materialdesignicons.min.css" rel="stylesheet">
     <link href="https://cdn.masastack.com/npm/materialicons/materialicons.css" rel="stylesheet">
     <link href="https://cdn.masastack.com/npm/fontawesome/v5.0.13/css/all.css" rel="stylesheet">
-    <link href="css/masa-blazor-pro.css" rel="stylesheet" />
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css" rel="stylesheet" />
```
Comment (doddgu) [inline src/Web/Masa.Mc.Web.Admin.Server/Pages/_Layout.cshtml:20]: Use MASA CDN, and why you need add animate?

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Web/Masa.Mc.Web.Admin.Server/Pages/_Layout.cshtml (hunk 3 above)
Comment (wzh425) [reply src/Web/Masa.Mc.Web.Admin.Server/Pages/_Layout.cshtml:20]: animate. CSS brings some auxiliary classes of animate

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**